### DISCUSSION TITLE
CUSA01867 - Shovel Knight: Treasure Trove

### DISCUSSION THREAD

**Discussion by Zehkarluz** [GoldHEN/GoldHEN_Cheat_Repository#10] [Cheat requests]
Inf health, magic or money. Please
Any other type of cheat are welcome too

**Comment by kmeps4** [comment]
Will check if there's a cheat available to add

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
